azure/datalake/store/lib.py

```diff
@@ -25,6 +25,14 @@
 import requests
 import requests.exceptions
 
+# this is required due to github issue, to ensure we don't lose perf from openPySSL: https://github.com/pyca/pyopenssl/issues/625
+try:
+    from requests.packages.urllib3.contrib.pyopenssl import extract_from_urllib3 as enforce_no_py_open_ssl
+    enforce_no_py_open_ssl()
```

Comment:
I am not quite sure I understand. You mean that I call enforce_no_py_open_ssl() in an `else` somewhere, or in an `if`?